packages/test/test-version-utils/src/describeCompat.ts

```diff
@@ -30,20 +31,38 @@ function createCompatSuite(
 				let provider: TestObjectProvider;
 				let resetAfterEach: boolean;
 				before(async function () {
-					provider = await getVersionedTestObjectProvider(
-						baseVersion,
-						config.loader,
-						{
-							type: driver,
-							version: config.driver,
-							config: {
-								r11s: { r11sEndpointName },
-								odsp: { tenantIndex },
+					try {
+						provider = await getVersionedTestObjectProvider(
+							baseVersion,
+							config.loader,
+							{
+								type: driver,
+								version: config.driver,
+								config: {
+									r11s: { r11sEndpointName },
+									odsp: { tenantIndex },
+								},
 							},
-						},
-						config.containerRuntime,
-						config.dataRuntime,
-					);
+							config.containerRuntime,
+							config.dataRuntime,
+						);
+					} catch (error) {
+						const logger = ChildLogger.create(
```

Comment:
It's a bit weird to create sub-logger for such short scope.
While it has its own negatives, I'd suggest using this line:
```ts
logger.sendErrorEvent(
    {
       eventName: "DescribeCompatSetup:TestObjectProviderLoadFailed",
       driverType: driver,
    },
    error,
);
```

Reply:
it's more for the `ITelemetryLoggerExt` methods than creating a child logger. I think the logger impl injected by `getTestLogger` is just an `ITelemetryBaseLogger`, which doesn't have `sendErrorEvent` for extracting the error props. This is probably more clear by putting the driver type on the event than the ChildLogger though, so I'll at least make that change.